.github/workflows/check-migrations.yml

```diff
@@ -0,0 +1,27 @@
+name: Check Migrations
+
+on:
+  pull_request:
+    types: [labeled, opened, synchronize, unlabeled]
+
+jobs:
+  check-migrations:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Rust Cache
+        uses: Swatinem/rust-cache@3cf7f8cc28d1b4e7d01e3783be10a97d55d483c8 # v2.7.1
+        with:
+          cache-on-failure: true
+
+      - name: Install submig
+        run: |
+          cargo install submig
+          submig --version
+
+      - name: Checkout repo
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 #v4.1.1
+
+      - name: Check migrations
+        continue-on-error: false
+        run: |
+          submig list .
```

Comment:
Cool! Would be nice to have a couple sentences explaining what this does :)

Reply:
I could add a link to https://crates.io/crates/submig :)

Reply:
>  however I think some of the migrations currently inside Unreleased tuples have been deployed already?

Yes indeed, the question is which ones.....

They don't really hurt being around (for now). Once the PR is merged, we can move all of the them to the next release and flush the list. Sure the content of `V1_05_00` will then be slightly incorrect in the sense that some migrations may have been introduced earlier.

I did create older version when the info was available as comment but it was mostly not the case.

Reply:
I added some doc and renamed also the GHA file to make it clearer.